packages/filebrowser/style/base.css

```diff
@@ -127,17 +132,20 @@
 |----------------------------------------------------------------------------*/
 
 .jp-DirListing {
-  flex: 1 1 auto;
-  display: flex;
-  flex-direction: column;
   outline: 0;
 }
 
 .jp-DirListing-header {
-  flex: 0 0 auto;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
+  display: grid;
+
+  /* TODO: Deduplicate with content grid */
+  grid-template-columns:
+    [checkbox] auto
```

Comment:
I wonder if we should prefix the names of areas like `jp-filebrowser-checkbox` to avoid conflicts.